hail/python/hailtop/hailctl/dev/benchmark/run/shuffle_benchmarks.py

```diff
@@ -0,0 +1,38 @@
+from os import path
+from tempfile import TemporaryDirectory, NamedTemporaryFile
+
+import hail as hl
+from .utils import benchmark, resource
+
+
+@benchmark
+def shuffle__key_rows_by_mt():
```

Comment:
why the double underscore?

Reply:
Ah, yeah I meant to ask you about this. Is it possible to use `-k` to select just a file? I only added the `shuffle__` so I could easily pick out all the functions in this file.

Reply:
hmm. I don't think that works at the moment. Can just do -k "^shuffle_"?

Reply:
yeah I can use a single underscore